airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -4,25 +4,38 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Github Source Spec",
     "type": "object",
-    "required": ["access_token", "repository", "start_date"],
+    "required": ["access_token", "start_date"],
     "additionalProperties": false,
     "properties": {
       "access_token": {
         "type": "string",
         "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
         "airbyte_secret": true
       },
+      "organization": {
+        "type": "string",
+        "examples": ["airbytehq/airbyte"],
+        "description": "Space-delimited list of GitHub organizations. e.g. `airbytehq singer-io`"
+      },
       "repository": {
         "type": "string",
         "examples": ["airbytehq/airbyte"],
-        "description": "GitHub repository. e.g. 'airbytehq/airbyte' or 'singer-io/tap-github'"
+        "description": "Space-delimited list of GitHub repositories, e.g. `airbytehq/airbyte singer-io/tap-github`"
       },
       "start_date": {
         "type": "string",
         "description": "The date from which you'd like to replicate data for GitHub in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated. Note that it will be used only in the following incremental streams: comments, commits and issues.",
         "examples": ["2021-03-01T00:00:00Z"],
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
       }
-    }
+    },
+    "anyOf": [
```

Comment:
can you use the format described here using `oneOf` instead of `anyOf`: https://docs.airbyte.io/connector-development/connector-specification-reference#using-oneofs